htdocs/admin/dict.php

```diff
@@ -1580,6 +1580,10 @@
 							{
 								$valuetoshow = $langs->trans($valuetoshow);
 							}
+							else
```

Comment:
Problem here is that we always translate value, and sometime we should just show the value as it is.
In which case did you get a missing translation ?

Reply:
well, after adding a new dictionary to my module, i noticed that the strings from my dictionary table are not translated when getting displayed on dolibarr admin/dict.php page.

also the value(s) are not always translated in this case, i mean especially not for external modules, you can check that on line
https://github.com/Dolibarr/dolibarr/blob/b49745e9890104f1f476a16a199d46df1540b624/htdocs/admin/dict.php#L1395
all the conditions after this line do not match with external modules, so the values are never translated for external modules.

Reply:
PS: i'm okay with you that sometimes we should just show the value as it is, but all the ways, $langs->trans() function will return the original string if no translation have been found.

The only other solution i can see here, is to implement the 'viewDictionaryFieldlist' hook function in my modules, but i'm not sure that this is really the best solution.